packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -7,24 +7,38 @@
 
 import fs from 'fs';
 import path from 'path';
+
 import { rollup } from 'rollup';
 import prettier from 'prettier';
 import lwcRollupPlugin from '@lwc/rollup-plugin';
+import { testFixtureDir } from 'jest-utils-lwc-internals';
 
-const FIXTURE_DIR = path.join(__dirname, 'fixtures');
+jest.setTimeout(10_000 /* 10 seconds */);
 
-const DIST_DIRNAME = 'dist';
-const MODULE_DIRNAME = 'modules';
-const FIXTURE_NAMESPACE = 'x';
+async function compileFixture({ input, dirname }) {
+    const modulesDir = path.resolve(dirname, './modules');
+    const outputFile = path.resolve(dirname, './dist/compiled.js');
 
-const CONFIG_FILENAME = 'config.json';
-const COMPILER_ENTRY_FILENAME = 'fixture.js';
-const EXPECTED_HTML_FILENAME = 'expected.html';
+    const bundle = await rollup({
+        input,
+        external: ['lwc'],
+        plugins: [
+            lwcRollupPlugin({
+                modules: [
+                    {
+                        dir: modulesDir,
+                    },
+                ],
+            }) as any,
```

Comment:
Works without the type assertion too. Didn't see any ts errors.
```suggestion
            }),
```